test/math_errhandling_tests.c

```diff
@@ -346,13 +346,19 @@ FLOAT_T makemathname(test_jn_snan)(void) { return makemathname(jn)(3, makemathna
 
 #endif
 
+#if INT_MAX == INT16_MAX
```

Comment:
20 should work on every target -- it only needs to be larger than `__DBL_MAX_EXP__` by a little bit :-)

Reply:
> 20 should work on every target -- it only needs to be larger than `__DBL_MAX_EXP__` by a little bit :-)

In other words, let's not have a special case for 16-bit int machines so that we can test the change before we have any of those in CI.

Reply:
That's much better, updated the commit with a smaller constant instead.